Dockerfile

```diff
@@ -17,11 +17,9 @@ RUN for PYTHON_VERSION in 2 3; do \
         NANSHE_VERSION=`conda list -f nanshe 2>/dev/null | \
                         tail -1 | \
                         python -c "from sys import stdin; print(stdin.read().split()[1])"` && \
```

Comment:
This should be dropped as well. Handling in PR ( https://github.com/nanshe-org/docker_nanshe/pull/39 ).